pyoxidizer/src/pyrepackager/packaging_rule.rs

```diff
@@ -839,24 +839,18 @@ fn resolve_pip_install_simple(
         pip_args.extend(rule.extra_args.clone().unwrap());
     }
 
-    // TODO send stderr to stdout.
-    let mut cmd = std::process::Command::new(&dist.python_exe)
-        .args(&pip_args)
-        .envs(&extra_envs)
-        .stdout(std::process::Stdio::piped())
-        .spawn()
+    let handler = cmd(&dist.python_exe, &pip_args)
+        .full_env(&extra_envs)
+        .stderr_to_stdout()
+        .stdout_capture()
+        .start()
         .expect("error running pip");
-    {
-        let stdout = cmd.stdout.as_mut().unwrap();
-        let reader = BufReader::new(stdout);
+    
+    let output = handler.wait().unwrap();
+    let stdout = &output.stdout;
+    warn!(logger, "{}", std::str::from_utf8(&stdout).unwrap());
```

Comment:
Can `duct` stream output? I don't like buffering all the output until the process finishes because it can suppress output from long-running commands until much later after it occurs. This gives the feeling of slow execution, makes it harder to understand timings, etc.

Reply:
They have an example of it in the docs, I'll make that work (probably tomorrow)